### PR TITLE
test: introduce unstable fuzz create table test

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

The unstable fuzz test for standalone will kill the GreptimeDB process randomly. The `unstable_fuzz_create_table` test tries to verify the atomicity of the create table procedure.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -24,3 +24,7 @@ GT_KAFKA_ENDPOINTS = localhost:9092
 
 # Setting for fuzz tests
 GT_MYSQL_ADDR = localhost:4002
+
+# Setting for usntable fuzz tests
+GT_FUZZ_BINARY_PATH=/path/to/
+GT_FUZZ_INSTANCE_ROOT_DIR=/tmp/unstable_greptime
```
Comment (WenyXu) [inline .env.example:30]:
```suggestion 
GT_FUZZ_INSTANCE_ROOT_DIR=/tmp/unstable_greptime

```

File: .github/workflows/develop.yml [hunk 2]
```diff
@@ -163,8 +163,62 @@ jobs:
...
+          CUSTOM_LIBFUZZER_PATH: /usr/lib/llvm-14/lib/libFuzzer.a
+          GT_MYSQL_ADDR: 127.0.0.1:4002
+          GT_FUZZ_BINARY_PATH: ./bins/greptime
+          GT_FUZZ_INSTANCE_ROOT_DIR: /tmp/unstable-greptime/
+        with:
+          target: ${{ matrix.target }}
+          max-total-time: 120
+          unstable: 'true'
+      - name: Upload unstable fuzz test logs
+        if: always()
```
Comment (WenyXu) [inline .github/workflows/develop.yml:215]:
```suggestion
        if: failure()
```

File: tests-fuzz/targets/unstable/fuzz_create_table_standalone.rs [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+struct FuzzInput {
+    seed: u64,
+    num: usize,
+}
+
+impl Arbitrary<'_> for FuzzInput {
+    fn arbitrary(u: &mut Unstructured<'_>) -> arbitrary::Result<Self> {
+        let seed = u.int_in_range(u64::MIN..=u64::MAX)?;
+        let mut rng = ChaChaRng::seed_from_u64(seed);
+        let num = rng.gen_range(1..100);
```
Comment (WenyXu) [inline tests-fuzz/targets/unstable/fuzz_create_table_standalone.rs:70]:
```suggestion
        let num = rng.gen_range(1..500);
```

File: .github/actions/fuzz-test/action.yaml [hunk 4]
```diff
@@ -3,11 +3,17 @@ description: 'Fuzz test given setup and service'
 inputs:
   target:
     description: "The fuzz target to test"
+    required: true
+  max-total-time:
+    description: "Max total time(secs)"
+    required: true
+  unstable:
+    default: 'false'
+    description: "enable unstable feature"
```
Comment (fengjiachun) [inline .github/actions/fuzz-test/action.yaml:12]:
```suggestion
    description: "Enable unstable feature"
```

File: tests-fuzz/src/utils.rs [hunk 5]
```diff
@@ -12,31 +12,74 @@
...
+        Some(addr)
+    } else {
+        info!("GT_MYSQL_ADDR is empty, ignores test");
+        None
+    };
+
+    init_greptime_connections(mysql).await
+}
+
+/// Connects to GreptimeDB via env variables.
```
Comment (fengjiachun) [inline tests-fuzz/src/utils.rs:46]: The doc seems does not correct

File: tests-fuzz/src/utils/config.rs [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+/// Returns rendered config file.
+pub fn render_config_file<C: Serialize>(template_path: &str, context: &C) -> String {
+    let mut tt = TinyTemplate::new();
+    let template = std::fs::read_to_string(template_path).unwrap();
+    tt.add_template(template_path, &template).unwrap();
+    tt.render(template_path, context).unwrap()
+}
+
+pub async fn write_config_file<C: Serialize>(
```
Comment (WenyXu) [inline tests-fuzz/src/utils/config.rs:42]:
```suggestion
// Writes config file to `output_path`.
pub async fn write_config_file<C: Serialize>(
```

File: tests-fuzz/README.md [hunk 7]
```diff
@@ -9,6 +9,25 @@ cargo install cargo-fuzz
...
+GT_FUZZ_BINARY_PATH = /path/to/
+```
+
+Change the instance root directory(the default value: `/tmp/unstable_greptime/`)
+```
+GT_FUZZ_INSTANCE_ROOT_DIR = /path/to/
+```
+
+
+
```
Comment (WenyXu) [inline tests-fuzz/README.md:30]:
```suggestion
```

**Review by fengjiachun — APPROVED** [review]
LGTM

File: .env.example [hunk 8]
```diff
@@ -24,3 +24,7 @@ GT_KAFKA_ENDPOINTS = localhost:9092
 
 # Setting for fuzz tests
 GT_MYSQL_ADDR = localhost:4002
+
+# Setting for usntable fuzz tests
```
Comment (tisonkun) [inline .env.example:28]:
```suggestion
# Setting for unstable fuzz tests
```

**Review by tisonkun — APPROVED** [review]

**Pull request merged**